packages/matter.js/src/session/pase/PaseServer.ts

```diff
@@ -46,13 +56,42 @@ export class PaseServer implements ProtocolHandler<MatterDevice> {
         try {
             await this.handlePairingRequest(exchange.session.getContext(), messenger);
         } catch (error) {
-            logger.error("An error occured during the commissioning", error);
-            await messenger.sendError();
+            this.pairingErrors++;
+            logger.error("An error occurred during the PASE commissioning.", error);
+
+            // if we got a ChannelStatusResponseError in we do not need to send the same our
```

Comment:
"send ourselves"?

Reply:
ok will rephrase ... means: if we received such an error we should not respond with the same error back, but just react without sending any answer